### DISCUSSION TITLE
Feature request. Use previous OS Baseline as a starting point for new Baseline

### DISCUSSION THREAD

**Discussion by rarmfield** [usnistgov/macos_security#433] [Ideas]
Now that Sequoia is available, I need to create a new baseline.  It would be nice to be able to build on the previous baselines created and add/change the new features rather than having to start from 0.  This would make sure that we do not inadvertently create a disparity between OSes.

**Comment by jmahlman** [comment]
You can already do this; just copy your custom rules that are created over to the new OS branch. This is what I do every single OS release.
